### PR TITLE
add ecmwf-opendata

### PR DESCRIPTION
**Pull request by raybellwaves**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by raybellwaves** [comment]
Friendly ping for a review please @conda-forge/staged-recipes

File: recipes/ecmwf-opendata/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/e/ecmwf-opendata/ecmwf-opendata-{{ version }}.tar.gz
+  sha256: 628891bd72af645616c257ca02d79e8a22baa12f49865d78d23b6459ce0ab056
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
```
Comment (BastianZim) [inline recipes/ecmwf-opendata/meta.yaml:14]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```

**Review by BastianZim — APPROVED** [review]

**Comment by BastianZim** [comment]
Please ping me here as well.

**Comment by raybellwaves** [comment]
Friendly ping @BastianZim

**Pull request merged**